.github/workflows/subca-clone-test.yml

```diff
@@ -0,0 +1,406 @@
+      - name: Set up root CA container
+        run: |
+          tests/bin/runner-init.sh
+        env:
+          IMAGE: pki-runner
+          NAME: root-ca
+          HOSTNAME: root-ca.example.com
+
+      - name: Connect root CA container to network
+        run: docker network connect example root-ca --alias root-ca.example.com
+
+      - name: Create root CA in NSS database
+        run: |
+          # https://github.com/dogtagpki/pki/wiki/Creating-Self-Signed-CA-Signing-Certificate-with-PKI-NSS
+
+          docker exec root-ca pki nss-cert-request \
+              --subject "CN=Root CA Signing Certificate" \
+              --ext /usr/share/pki/server/certs/ca_signing.conf \
+              --csr $PKIDIR/root-ca_signing.csr
+          docker exec root-ca pki nss-cert-issue \
+              --csr $PKIDIR/root-ca_signing.csr \
+              --ext /usr/share/pki/server/certs/ca_signing.conf \
+              --cert $PKIDIR/root-ca_signing.crt
+          docker exec root-ca pki nss-cert-import \
+              --cert $PKIDIR/root-ca_signing.crt \
+              --trust CT,C,C \
+              root-ca_signing
+
+      - name: Set up primary DS container
+        run: |
+          tests/bin/runner-init.sh
+        env:
+          IMAGE: pki-runner
+          NAME: primary-ds
+          HOSTNAME: primary-ds.example.com
+
+      - name: Connect primary DS container to network
+        run: docker network connect example primary-ds --alias primary-ds.example.com
+
+      - name: Install DS package
+        run: docker exec primary-ds dnf install -y 389-ds-base
+
+      - name: Install primary DS instance
+        run: docker exec primary-ds $PKIDIR/tests/bin/ds-create.sh
+
+      - name: Set up primary sub-CA container
+        run: |
+          tests/bin/runner-init.sh
+        env:
+          IMAGE: pki-runner
+          NAME: primary-subca
+          HOSTNAME: primary-subca.example.com
+
+      - name: Connect primary sub-CA container to network
+        run: docker network connect example primary-subca --alias primary-subca.example.com
+
+      - name: Install primary sub-CA (step 1)
+        run: |
+          # docs/installation/ca/Installing_CA_with_External_CA_Signing_Certificate.md
+
+          # customize installation config file
+          docker exec primary-subca sed \
+              -e "s,^\(pki_ca_signing_csr_path\)=.*$,\1=$PKIDIR/subca_signing.csr," \
+              -e "$ a pki_ds_hostname=primary-ds.example.com" \
+              -e "$ a pki_client_admin_cert_p12=$PKIDIR/caadmin.p12" \
+              /usr/share/pki/server/examples/installation/ca-external-cert-step1.cfg \
+              | tee ca-external-cert-step1.cfg
+
+          docker exec primary-subca pkispawn \
+              -f $PKIDIR/ca-external-cert-step1.cfg \
+              -s CA \
+              -v
+
+      - name: Issue primary sub-CA signing cert
+        run: |
+          # https://github.com/dogtagpki/pki/wiki/Creating-Self-Signed-CA-Signing-Certificate-with-PKI-NSS
```

Comment:
is this the right link?  It looks to be the same link as the self-sign ca above, which only shows how to do self-sign, not with --issuer

Reply:
Sorry, it should have been this link: https://github.com/dogtagpki/pki/wiki/Issuing-CA-Signing-Certificate-with-PKI-NSS.